### DISCUSSION TITLE
Help needed with setting up on Raspberry Pi 5 with bluetooth audio on RaspberryPi OS

### DISCUSSION THREAD

**Discussion by nookied** [mikebrady/shairport-sync#1872] [Q&A]
Hi,

I'm struggling with setting up a shairport-sync server on Raspberry Pi 5 running latest Raspberry Pi OS (with GUI) over Bluetooth.
I followed the instructions in [Build and Install Shairport Sync](https://github.com/mikebrady/shairport-sync/blob/master/BUILD.md), tried building with alsa, tried setting up Pulse Audio with [this tutorial](https://github.com/mikebrady/shairport-sync/blob/master/ADVANCED%20TOPICS/PulseAudioAndPipeWire.md), but haven't been able to output the audio from airplay to the Bluetooth device.

The Bluetooth device is paired, connected, and plays audio, both from GUI and through `aplay` CLI command.
The airplay device is discoverable and connectable, but no audio stream comes through to the Bluetooth speaker.

Could anyone please help me out with this? I'm still relatively new to Linux systems and might not have understood all the research I have done on that topic...

**Comment by mikebrady** [comment]
Thanks for the post. Some of the closed issues refer to users trying to get Bluetooth working with Shairport Sync.
